generated/nixnetsocket/nixnetsocket.proto

```diff
@@ -233,11 +250,11 @@ message SelectResponse {
 message SetSockOptRequest {
   nidevice_grpc.Session socket = 1;
   int32 level = 2;
-  SockOptData opt_data = 3;
```

Comment:
Note: Any noise with SetSockOpt is because I noticed I should have moved the opt_data field to the end after optname so I corrected that with these changes.